### DISCUSSION TITLE
Need help with resolving deploy error..

### DISCUSSION THREAD

**Discussion by chainismm** [vercel/community#2366] [Help]
Resolved

**Comment by amyegan** [comment]
Hi @ArbHouse! There are a few things that could be causing the error.

- Did you use an older Node version prior to 18 for development? If so, try changing the [Node.js Version](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version) in your project settings to match your dev version.
- Do you have the `engines` property set in package.json? If so, make sure it is set to `14.x`,  `16.x`, or `18.x` without `>` or any other extra characters.

If none of that works, please share a minimal reproduction that I can use to get a better look at the problem.

**Reply by amyegan** [reply]
Oh! You seem to have fixed it before I could post my response. Would you mind sharing the solution in case it helps someone else?
